src/esp/assets/GenericInstanceMeshData.cpp

```diff
@@ -262,6 +265,15 @@ void GenericInstanceMeshData::uploadBuffersToGPU(bool forceReload) {
       .setIndexBuffer(std::move(indices), 0,
                       Mn::GL::MeshIndexType::UnsignedInt);
 
+  cpu_ibo_ = new_indices;
+
+  collisionMeshData_.positions =
```

Comment:
We are setting collisionMeshData_ already in loadPly and now here also. I think it may be possible to only set collisionMeshData_ in one place by moving some of the logic in this function to loadPly. It also feels like this a bug fix that should really be its own change.

Reply:
Good point. I've create a new PR #389 for this including your previous suggestion above. 👍